### PR TITLE
add support for form/yaml switcher for channels

### PR DESCRIPTION
**Pull request by sahil143**
fixes: https://issues.redhat.com/browse/ODC-5340

This PR adds support of form YAML switcher for channel creation

Kapture 2021-11-16 at 15 10 54

### CODE REVIEW

File: frontend/packages/knative-plugin/src/components/add/channels/ChannelForm.tsx [hunk 1]
```diff
@@ -44,32 +55,45 @@ const ChannelForm: React.FC<FormikProps<FormikValues> & OwnProps> = ({
   const [defaultConfiguredChannel, defaultConfiguredChannelLoaded] = useDefaultChannelConfiguration(
     namespace,
   );
-  const channelHasFormView = values.type && isDefaultChannel(getChannelKind(values.type));
-  const channelKind = getChannelKind(values.type);
+  // const channelHasFormView = values.type && isDefaultChannel(getChannelKind(values.type));
```
Comment (karthikjeeyar) [inline frontend/packages/knative-plugin/src/components/add/channels/ChannelForm.tsx:58]:
```suggestion
```

**Comment by invincibleJai** [comment]
/approve

verified the changes, looks good. Thanks!!

**Comment by sahil143** [comment]
Propagate from epic:

/label docs-approved
/label px-approved

**Pull request merged**